tests/scripts/all.sh

```diff
@@ -323,6 +327,9 @@ cleanup()
             cp -p "$x$backup_suffix" "$x"
         fi
     done
+
+    # Restrore CFLAGS
+    export CFLAGS="${GLOBAL_CFLAGS}"
```

Comment:
This is pointless since each component runs in its own subshell. `cleanup` must clean up files but does not need to do anything about the internal state of the shell.